packages/terra-data-grid/src/utils/WorklistDataGridUtils.js

```diff
@@ -8,8 +8,16 @@ const ROW_SELECTION_COLUMN = {
   isResizable: false,
 };
 
+const TABLE_ROW_SELECTION_COLUMN = {
+  id: 'table-rowSelectionColumn',
+  width: 40,
+  isSelectable: true,
+  isResizable: false,
+};
+
 const WorklistDataGridUtils = {
   ROW_SELECTION_COLUMN,
+  TABLE_ROW_SELECTION_COLUMN,
 };
```

Comment:
We do because we're using different ID's for the row selection column in each component. Table uses `table-rowSelectionColumn` while WDG uses `worklistDataGrid-rowSelectionColumn`.

Reply:
You don't need this entire structure just for an id.